files/ko/web/http/headers/referer/index.html

```diff
@@ -66,12 +73,16 @@ <h2 id="명세서">명세서</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
+<h2 id="Browser_compatibility">브라우저 호완성</h2>
 
-<p>{{Compat("http.headers.Referer")}}</p>
+<p>{{Compat}}</p>
 
-<h2 id="함께_참고할_내용">함께 참고할 내용</h2>
+<h2 id="See_also">같이 보기</h2>
 
 <ul>
- <li>{{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}</li>
-</ul>
+  <li>{{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}</li>
+  <li>The obsolete {{HTTPHeader("Content-Security-Policy")}} {{HTTPHeader("Content-Security-Policy/referrer", "referrer")}} {{deprecated_inline}} directive.</li>
+  <li><a href="/en-US/docs/Web/Security/Same-origin_policy">Same-origin policy</a></li>
+  <li><a href="https://blog.mozilla.org/security/2015/01/21/meta-referrer/">Tighter Control Over Your Referrers – Mozilla Security Blog</a></li>
+ </ul>
+
```

Comment:
Fetch 관련 링크가 빠져있는 것 같아 추가하고 번역되지 않은 The obsolete directive 부분 추가하였습니다. obsolete라는 표현의 번역은 다양할 수 있겠으나 사전 정의인 '구식'은 조금 어감이 이상하여 outdated된으로 표기해두었습니다. 다른 적절한 번역이 있다면 자유롭게 제안 주세요! 
그 외, 링크 패스를 ko로 수정하였습니다.

```suggestion
  <li>{{interwiki("wikipedia", "HTTP_referer", "HTTP referer on Wikipedia")}}</li>
  <li><a href="/ko/docs/Web/API/Fetch_API">Fetch</a>: {{domxref("Request.referrerPolicy")}}</li>
  <li>outdated 된 {{HTTPHeader("Content-Security-Policy")}} {{HTTPHeader("Content-Security-Policy/referrer", "referrer")}} {{deprecated_inline}} 디렉티브</li>
  <li><a href="/ko/docs/Web/Security/Same-origin_policy">Same-origin policy</a></li>
  <li><a href="https://blog.mozilla.org/security/2015/01/21/meta-referrer/">Tighter Control Over Your Referrers – Mozilla Security Blog</a></li>
```